frontend/web/components/FlagOwnerGroups.js

```diff
@@ -0,0 +1,117 @@
+import React, { Component } from 'react'
+import data from 'common/data/base/_data'
+import UserSelect from './UserSelect'
+import ConfigProvider from 'common/providers/ConfigProvider'
+import Icon from './Icon'
+import { close } from 'ionicons/icons'
+import { IonIcon } from '@ionic/react'
+import GroupSelect from './GroupSelect'
+import { getProjectFlag } from 'common/services/useProjectFlag'
+import { getStore } from 'common/store'
+
+class TheComponent extends Component {
+  state = {}
+  componentDidMount() {
+    this.getData()
+  }
+
+  getData = () => {
+    getProjectFlag(getStore(), {
+      id: this.props.id,
+      project: this.props.projectId,
+    }).then((res) => {
+      const owners = (res.data.group_owners || []).map((v) => v.id)
+      this.setState({ owners })
```

Comment:
Since we already have `owners` for normal users I think here and elsewhere the naming should be spelled out as `group_owners` since it may cause confusion with `owners`.

Reply:
Yep sure makes sense